### PR TITLE
Update client package documentation.

### PR DESCRIPTION
**Pull request by petermattis**

### CODE REVIEW

File: client/doc.go [hunk 1]
```diff
@@ -16,107 +16,111 @@
...
-
-The KV client is a fully-featured client of Cockroach's key-value
-database. It provides a simple, synchronous interface well-suited to
-parallel updates and queries.
+The DB client is a fully-featured client of Cockroach's key-value database. It
+provides a simple, synchronous interface well-suited to parallel updates and
+queries.
 
 The simplest way to use the client is through the Run method. Run
 synchronously invokes the call and fills in the the reply and returns
```
Comment (tschottdorf) [inline client/doc.go]: invokes the call, filles in the reply, and

File: client/doc.go [hunk 2]
```diff
@@ -16,107 +16,111 @@
...
-Note that with Cockroach's lock-free transactions, clients should
-expect retries as a matter of course. This is why the transaction
-functionality is exposed through a retryable function. The retryable
-function should have no side effects which are not idempotent.
-
-Transactions should endeavor to write using KV.Prepare calls. This
-allows writes to the same range to be batched together. In cases where
-the entire transaction affects only a single range, transactions can
-commit in a single round trip.
+	db, err := client.Open("https://localhost:8080")
```
Comment (tschottdorf) [inline client/doc.go]: does this need `root@` or something like that?

File: client/doc.go [hunk 3]
```diff
@@ -16,107 +16,111 @@
...
+	}
+	if gr, err := db.Get("a"); err != nil {
+		log.Fatal(err)
+	} else {
+		log.Printf("%s", gr.ValueBytes())  // "hello"
+	}
+
+The API is synchronous, but accommodates efficient parallel updates and queries
+using Batch objects. An arbitrary number of calls may be added to a Batch which
+are executed using DB.Run. Note however that the individual calls within a
```
Comment (tschottdorf) [inline client/doc.go]: s/are/is/

**Comment by tschottdorf** [comment]
LGTM

File: client/doc.go (hunk 1 above)
Comment (petermattis) [inline client/doc.go]: Done.

File: client/doc.go (hunk 2 above)
Comment (petermattis) [inline client/doc.go]: `root@` is the default user, but I suppose being explicit here is useful.

File: client/doc.go (hunk 3 above)
Comment (petermattis) [inline client/doc.go]: Done.

File: client/doc.go [hunk 4]
```diff
@@ -16,107 +16,111 @@
...
+	max := []byte(nil)
+	for _, row := range acResult.Rows {
+		if bytes.Compare(max, row.ValueBytes()) < 0 {
+			max = row.ValueBytes()
+		}
+	}
+
+	b2 := &client.Batch{}
+	for _, row := range xzResult.Rows {
+		b2.Put(row.Key, bytes.Join([][]byte{kv.ValueBytes(), max}, []byte(nil)))
```
Comment (petermattis) [inline client/doc.go]: Also: `s/kv./row./g`

**Pull request merged**